content/en/docs/components/trainer/operator-guides/runtime.md

```diff
@@ -99,157 +99,86 @@ spec:
-                  limits:
-                    nvidia.com/gpu: "1"
-```
+| Runtime Name          | Pre-trained LLM |
+| --------------------- | --------------- |
+| torchtune-llama3.2-1b | Llama 3.2 (1B)  |
+| torchtune-llama3.2-3b | Llama 3.2 (3B)  |
 
-### Ancestor Label Requirements for ReplicatedJobs
+### Runtime Deprecation Policy
 
-When defining `replicatedJobs` such as `dataset-initializer`, `model-initializer`, and `node`,
-it is important to ensure that each job template includes the necessary ancestor labels.
-These labels are used by the Kubeflow Trainer controller to inject values from the TrainJob to
-the underlying training job.
+As ML frameworks evolve over time, the Kubeflow community may decide to deprecate
+certain supported ClusterTrainingRuntimes. To avoid breaking existing users, we follow a deprecation
+policy before removing any Runtime.
 
-**Required Labels:**
+A supported ClusterTrainingRuntime may be marked as deprecated and is eligible for removal starting
+from **two minor releases** after its deprecation.
 
-- `trainer.kubeflow.org/trainjob-ancestor-step`: Specifies the role or step of the replicated job in the training workflow (e.g., `dataset-initializer`, `model-initializer` or `trainer`).
+These measures are taken to inform users about Runtime deprecation:
 
-**Example:**
+- Add the following label to the deprecated Runtime:
 
-```YAML
-apiVersion: kubeflow.org/v2alpha1
-kind: ClusterTrainingRuntime
-metadata:
-  name: example-runtime
-spec:
-  template:
-    spec:
-      replicatedJobs:
-        - name: dataset-initializer
-          template:
-            metadata:
-              labels:
-                trainer.kubeflow.org/trainjob-ancestor-step: dataset-initializer
-            spec:
-              template:
-                spec:
-                  containers:
-                    - name: dataset-initializer
-                      image: ghcr.io/kubeflow/trainer/dataset-initializer
-        - name: model-initializer
-          dependsOn:
-            - name: dataset-initializer
-              status: Complete
-          template:
-            metadata:
-              labels:
-                trainer.kubeflow.org/trainjob-ancestor-step: model-initializer
-            spec:
-              template:
-                spec:
-                  containers:
-                    - name: model-initializer
-                      image: ghcr.io/kubeflow/trainer/model-initializer
-        - name: node
-          dependsOn:
-            - name: model-initializer
-              status: Complete
-          template:
-            metadata:
-              labels:
-                trainer.kubeflow.org/trainjob-ancestor-step: trainer
-            spec:
-              template:
-                spec:
-                  containers:
-                    - name: node
-                      image: ghcr.io/kubeflow/trainer/torchtune-trainer
-```
+  ```yaml
+  trainer.kubeflow.org/deprecated: "true"
+  ```
```

Comment:
@astefanutti @tenzen-y I introduced the deprecated label to the runtimes.
Does it look good ?